### PR TITLE
[FIX] Get Install Info failing because of WebGL platform

### PR DESCRIPTION
**Pull request by flavioislima**
Fix issue that is making games like The Unfettered: Awakening not loading info on the Game Page because it received `webgl` as game platform but the client doesn't understand it.
Other games that have a `WebGL` platform on the dev portal might be affect as well.

### How to Test

So Far, only game affected is The Unfettered: Awakening.

Simply add the game to your library and try to go to the game page.
On `main` it will show an error screen:
image

On This branch it should load the page just fine:
image

Installation should work just fine as well.

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by eliobricenov — APPROVED** [review]

File: src/frontend/screens/Game/GamePage/index.tsx [hunk 1]
```diff
@@ -232,9 +232,18 @@ export default observer(function GamePage(): JSX.Element | null {
               setGameInstallInfo(info)
             })
             .catch((error) => {
-              console.error(error)
-              window.api.logError(`${`${error}`}`)
-              setHasError({ error: true, message: `${error}` })
+              const errorMessage = t(
```
Comment (flavioislima) [inline src/frontend/screens/Game/GamePage/index.tsx:235]: Improved the error message here for better debugging next time the GamePage fails to load.

File: src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx [hunk 2]
```diff
@@ -412,7 +412,11 @@ export default function DownloadDialog({
     return t('button.no-path-selected', 'No path selected')
   }
 
-  const isWebGame = gameInstallInfo?.game['name'] === 'web'
+  const isWebGame =
+    gameInstallInfo?.game['name'] === 'web' ||
+    gameInstallInfo?.game['name'] === 'webgl' ||
+    platformToInstall === 'Browser'
```
Comment (BrettCleary) [inline src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx:418]: webgl platform is not a web game. we should remove that line

File: src/frontend/screens/Library/components/GameCard/index.tsx [hunk 3]
```diff
@@ -165,7 +165,10 @@ const GameCard = ({
 
   const isHiddenGame = libraryState.isGameHidden(appName)
   const isBrowserGame =
-    installPlatform === 'Browser' || installPlatform === 'web'
+    installPlatform === 'Browser' ||
+    installPlatform === 'web' ||
+    // @ts-expect-error will need to update valist SDK first
+    installPlatform === 'webgl'
```
Comment (BrettCleary) [inline src/frontend/screens/Library/components/GameCard/index.tsx:171]: webgl is not a browser game in this context. it is a download url to a webgl build of a game. it should not be surfaced in the client

**Review by BrettCleary — CHANGES_REQUESTED** [review]
Getting this in the logs

```
Could not launch web game for 0xc76153ec6edc0c42cc73433bbbe0b072581aa2e23930604fb25fba52f7dee14b
```

We shouldn't try to launch webgl platform as a web game

File: src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx (hunk 2 above)
Comment (flavioislima) [reply src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx]: fixed

File: src/frontend/screens/Library/components/GameCard/index.tsx (hunk 3 above)
Comment (flavioislima) [reply src/frontend/screens/Library/components/GameCard/index.tsx]: fixed

**Comment by flavioislima** [comment]
Fixed all of those @BrettCleary thanks for testing and finding those bugs.

**Review by BrettCleary — APPROVED** [review]
nice. code LGTM and passed all my manual tests

**Review by nyghtstalker — APPROVED** [review]
I am able to view the library game page and install both `Main` and `closed_test` branches.

image

**Pull request merged**